src/main/java/org/folio/rest/converter/titles/TitleConverter.java

```diff
@@ -100,10 +103,34 @@
     return title;
   }
 
+  private List<CustomerResources> getSortedCustomerResourcesList(List<CustomerResources> customerResourcesList) {
+    if (customerResourcesList != null) {
```

Comment:
do we have to check customerResourcesList != null twice? https://github.com/folio-org/mod-kb-ebsco-java/blob/5ad25779e56d2edcf786959ffdcbe5481b3415ee/src/main/java/org/folio/rest/converter/titles/TitleConverter.java#L82

Reply:
Agree